.circleci/config.yml

```diff
@@ -1,10 +1,18 @@
-version: 2
-
-references:
-  container_config: &container_config
+version: 2.1
+executors:
+  infrastructure_container_unstable:
+    docker:
+      - image: aeternity/infrastructure:master
+  infrastructure_container_stable:
+    docker:
+      - image: aeternity/infrastructure:stable
```

Comment:
Should https://github.com/aeternity/infrastructure/pull/266 not be present in stable branch also?

Reply:
yes